master/buildbot/statistics/storage_backends/base.py

```diff
@@ -0,0 +1,27 @@
+# This file is part of Buildbot.  Buildbot is free software: you can
+# redistribute it and/or modify it under the terms of the GNU General Public
+# License as published by the Free Software Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright Buildbot Team Members
+
+from twisted.internet import defer
+
+
+class StatsStorageBase(object):
+
+    """
+    Base class for sub service responsible for passing on stats data to
+    a storage backend
+    """
+
+    def thd_postStatsValue(self, post_data, series_name, context=None):
+        return defer.succeed(None)
```

Comment:
I did add abc here for the next PR: https://github.com/prasoon2211/buildbot/commit/4b436a997649529f021e5312423cbe5a953139c1

So, this can be merged as is and we can apply next PR to correct this.